cloudify_agent/resources/pm/shared/broker.crt.template

```diff
@@ -0,0 +1 @@
+{{ cert }}
```

Comment:
if all you are doing is replacing a single value, then there really isn't any need of using a template in this case is there? simply write the content of `cert` into the file directly